api_v3/services/UserEntryService.php

```diff
@@ -51,9 +51,17 @@ public function addUserEntryImpl($dbUserEntry)
 	 */
 	public function updateAction($id, KalturaUserEntry $userEntry)
 	{
+		if (isset($userEntry->status))
+		{
+			UserEntryPeer::setUseCriteriaFilter(false);
```

Comment:
instead of enable/disable the default criteria you can just use retrieveByPKNoFilter